app/views/account/pages/_pages_table.html.slim

```diff
@@ -1,5 +1,5 @@
 tbody
   tr
-    td =link_to span= page.title, account_page_path(page)
+    td = link_to span = page.title, account_company_page_path(page.company, page.id)
```

Comment:
no need to pass more than 1 AR instance for resourceful helper
`account_company_page_path(page)` - should be enough

Reply:
https://www.dropbox.com/s/29rlovompqjvoiz/Screenshot%202017-12-27%2021.28.44.png?dl=0